product-scenarios/scenarios-common/src/main/java/org/wso2/am/scenario/test/common/ScenarioTestConstants.java

```diff
@@ -21,7 +21,7 @@
     public static final String SUBSCRIBER_ROLE  = "Internal/subscriber";
     public static final String EVERYONE_ROLE = "Internal/everyone";
     public static final String TENANT_WSO2  = "wso2.com";
-    public static final long TIMEOUT_API_APPEAR_IN_STORE_AFTER_PUBLISH = 25000;
+    public static final long TIMEOUT_API_APPEAR_IN_STORE_AFTER_PUBLISH = 30000;
```

Comment:
Shall we increase this to 40000

Reply:
https://github.com/wso2/product-apim/pull/8242/commits/d1d1f3fd3d2b4bc76716fb25ace45912debada89